app/src/features/projects/components/ProjectFundingForm.test.tsx

```diff
@@ -0,0 +1,134 @@
+import { IMultiAutocompleteFieldOption } from 'components/fields/MultiAutocompleteFieldVariableSize';
+import { Formik } from 'formik';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import ProjectFundingForm, {
+  IInvestmentActionCategoryOption,
+  IProjectFundingForm,
+  ProjectFundingFormInitialValues,
+  ProjectFundingFormYupSchema
+} from './ProjectFundingForm';
+
+const first_nations: IMultiAutocompleteFieldOption[] = [
+  {
+    value: 1,
+    label: 'nation 1'
+  },
+  {
+    value: 2,
+    label: 'nation 2'
+  }
+];
+
+const stakeholder_partnerships: IMultiAutocompleteFieldOption[] = [
+  {
+    value: 1,
+    label: 'partner 1'
+  },
+  {
+    value: 2,
+    label: 'partner 2'
+  }
+];
+
+const funding_sources: IMultiAutocompleteFieldOption[] = [
+  {
+    value: 1,
+    label: 'agency 1'
+  },
+  {
+    value: 2,
+    label: 'agency 2'
+  },
+  {
+    value: 3,
+    label: 'agency 3'
+  }
+];
+
+const investment_action_category: IInvestmentActionCategoryOption[] = [
+  {
+    value: 1,
+    fs_id: 1,
+    label: 'action 1'
+  },
+  {
+    value: 2,
+    fs_id: 2,
+    label: 'category 1'
+  },
+  {
+    value: 3,
+    fs_id: 3,
+    label: 'not applicable'
+  }
+];
+
+describe('ProjectFundingForm', () => {
+  it('renders correctly with default empty values', () => {
+    const tree = renderer
```

Comment:
question:
why not use the testing library here? like the fragment instead of react-test-renderer

Reply:
Lack of familiarity with the tools, mainly.

I tried to google what the difference between the 2 libraries was, and all I found was someone mentioning that `react-test-renderer` was for snapshot tests, and the `@testing-library/react` was for regular tests.  Maybe thats not the case though?

Reply:
hmm yea that doesn't seem right
the testing library is quite powerful and I think we should use it throughout this project, it will enable all sorts of tests for our components (snapshot and functionality)

Reply:
we have tests in the codebase which use the testing library to do snapshot tests using fragments:
```
const { asFragment } = render(<MyComponent />);
expect(asFragment()).toMatchSnapshot();
```

Reply:
Yeah if the @testing-library/react does everything the other library does, and more, then no argument from me.

Presumably that other library has some purpose? Or maybe it used to and its been supplanted by the @testing-library/react?

Reply:
yeah I think it does everything enzyme/other libraries allow us to do :)

Reply:
The only difference I could find is that the react-test-renderer renders using the virtual ReactDOM, and the @testing-library/react renders using a the regular DOM.

No one commented on whether or not it mattered though, so :/